### ISSUE TITLE
Design system: tables

### ISSUE THREAD

**Issue by greatislander** [accessibility-exchange/platform#378]

**Comment by jobara** [comment]
@greatislander and @cherylhjli are there designs for this?

**Comment by greatislander** [comment]
Yep, they are in the "Links and Buttons" section of the Main Design System sheet in Figma: https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1134%3A11567&t=lNoMoU04cqpPU3FY-4

**Comment by greatislander** [comment]
Overflow/horizontal scrolling affordances: https://looseleaf.netlify.app/components/detail/table--default.html
